packages/editor/src/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -420,6 +426,19 @@ export function HierarchicalTermSelector( { slug } ) {
 					onChange={ setFilter }
 				/>
 			) }
+			{ loading && (
+				<FlexItem
+					display="flex"
+					style={ {
+						justifyContent: 'center',
+						alignItems: 'center',
+						// Match SearchControl height to prevent layout shift.
+						height: '40px',
+					} }
+				>
+					<Spinner />
+				</FlexItem>
```

Comment:
```suggestion
				<Flex
					justify="center"
					style={ {
						// Match SearchControl height to prevent layout shift.
						height: '40px',
					} }
				>
					<Spinner />
				</Flex>
```

Nit: Can we use the `Flex` component directly?